megamek/src/megamek/common/TankTrailerHitch.java

```diff
@@ -0,0 +1,291 @@
+/*
+ * MegaMek - Copyright (C) 2002,2003,2004,2005 Ben Mazur (bmazur@sev.org)
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation; either version 2 of the License, or (at your option) any later
+ * version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ */
+
+package megamek.common;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Vector;
+
+/**
+ * Represents a trailer hitch that allows a wheeled or tracked vehicle to tow trailers.
+ *
+ * @see megamek.common.MechFileParser#postLoadInit
+ */
+
+public class TankTrailerHitch implements Transporter {
```

Comment:
Add the @Override annotation to methods that implement the interface.

Reply:
Fixed.